sdk/storage/storage-blob/src/Pipeline.ts

```diff
@@ -142,14 +129,14 @@ export class Pipeline {
  * @interface StoragePipelineOptions
  */
 export interface StoragePipelineOptions {
-  proxy?: ProxySettings | string;
+  proxyOptions?: ProxySettings;
```

Comment:
Should this be `ProxyOptions`?

Reply:
@XiaoningLiu, @jiacfan With this change, we no longer support passing of the proxy url directly to StoragePipelineOptions. User will have to provide the host, port, username, password separately.

If they want to use a url, then they can run the `getDefaultProxySettings()` to get the proxy settings and then pass it in the StoragePipelineOptions.

Like @bterlson said, in the future we can look into supporting url directly if needed.

Reply:
It's now ProxySettings (fwiw they are the same interface).

Reply:
Sure.